### DISCUSSION TITLE
Webhook gollum event not showing up on Discord

### DISCUSSION THREAD

**Discussion by fefu-marquez** [community/community#24729] [Apps, API and Webhooks]
<p>Hi everyone! I’m trying to set up a webhook in my Discord server exclusively for wiki updates, and it doesn’t seem to be working. I am getting a 204 response, but it doesn’t show any message in the server.</p>
<p>I tested adding the gollum event in an already working webhook, and nothing. After doing some digging through the internet, and not finding anything about it, I decided to open a topic and see if someone knows something about this and how to solve it.</p>
<p>Thanks!</p>

**Comment by nethgato** [comment]
<p>Hey there <a class="mention" href="/armatorx">@ArmatorX</a> :wave:</p>
<p>Are you able to let us know which repository this is? If it’s private, no worries! Though knowing which repo it is would allow me to take a peek at push logs for the wiki updates you’ve tried to send hook updates through.</p>
<p>Though what I would recommend is avoiding Discord to start, and using something like Requestbin:</p>
<p><a href="https://requestbin.com/" class="onebox" target="_blank" rel="noopener">https://requestbin.com/</a></p>
<p>…to create a temporary endpoint to test gollum against.</p>
<p>Should also reveal any errors you might be seeing. Once you get your Requestbin endpoint functional, try Discord again.</p>
<p>If you continue to run into problems, let us know some more details from testing and we can help from there!</p>

**Comment by fefu-marquez** [comment]
<p>Hello! Sorry for the late reply. The repo is this one: <a href="https://github.com/ArmatorX/dreamwalker" rel="noopener nofollow ugc">https://github.com/ArmatorX/dreamwalker</a>.</p>
<p>I’ll try the temporary endpoint thing later and get back to let you know how it went.</p>

**Comment by fefu-marquez** [comment, marked as answer]
<p>Okay. Today I looked into this a bit deeper in order to connect a few repositories with Discord that weren’t working. So now I have a bit of a better understanding about it. I thought that by just creating a webhook in Discord, and linking it to the repository, it should just magically work, but the parameters that Discord expects are not the same that GitHub sends.</p>
<p>So… yeah. After checking <a href="https://requestbin.com/" rel="noopener nofollow ugc">https://requestbin.com/</a>, I found <a href="https://pipedream.com/" rel="noopener nofollow ugc">https://pipedream.com/</a> and after fidgeting with it for a while, I managed to connect both ends. Highly recommended for anyone that has this or a similar problem, since it allows you to customize the message that you want to send, and which data you want to show.</p>
<p>Lastly, I want to highlight that somehow, when I first encountered this problem, I managed to show some messages on Discord, like commits. But appart from that most of the webhook requests were rejected. I don’t know why this is, if it’s from Discord end, or if it’s intended even, but it was confusing.</p>

**Comment by ali1234** [comment]
I just ran into the same problem. My webhook sends commit and issue events but wiki edits don't show up. The log shows that discord accepted the event with 204 but it doesn't do anything with it. I suspect the problem is on discord's side and it simply doesn't understand how to handle the request payload from wiki events.

Here's a link to discord's feedback section where someone listed all the github events that don't work:

https://support.discord.com/hc/en-us/community/posts/4420841341335-Improve-GitHub-webhooks-with-new-event-types
